src/components/Header/Header.jsx

```diff
@@ -2,20 +2,23 @@ import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import './Header.css';
 
-const Header = ({ children, isAdoptPage }) => {
+const Header = ({ children, isAdoptPage, isAboutPage }) => {
   return (
     <header>
       <Link className='app-title-link' to='/'>
         <h1 className='app-title font-face-modesto-condensed'>Fantastical Foundlings</h1>
       </Link>
-      {children}
+      <div className='button-container'>
+        {children}
+      </div>
     </header>
   );
 };
 
 Header.propTypes = {
-  children: PropTypes.element,
-  isAdoptPage: PropTypes.bool.isRequired
+  children: PropTypes.arrayOf(PropTypes.element),
+  isAdoptPage: PropTypes.bool.isRequired,
```

Comment:
i thought that for jsx you didnt need to use proptypes anymore?

Reply:
@Jesuitman So, my understanding is that for any React components (files containing JSX), you should use the extension `.jsx`, and for any files that are pure JavaScript (no JSX), you use `.js`.  You get to drop PropTypes if you're working with something like TypeScript, which would use `.tsx` and `.ts` files.